src/Kernel32.Desktop/Kernel32+INPUT_EVENT_RECORD.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="INPUT_EVENT_RECORD"/> nested type.
+    /// </content>
+    public partial class Kernel32
+    {
+        /// <summary>
+        /// The input event information. The format of this member depends on the event type specified by the <see cref="INPUT_RECORD.EventType"/> member.
+        /// </summary>
+        [StructLayout(LayoutKind.Explicit)]
+        public struct INPUT_EVENT_RECORD
```

Comment:
I'm afraid this struct must be deleted and its members moved inline into `INPUT_RECORD`. We tried this the way you have it here before with the `INPUT` type, and in practice it's pretty broken, as we discovered when writing unit tests for it due to the subtle behavior of struct copying when working with it in local variables.